overrides/config/ModernWarfare.cfg

```diff
@@ -11,7 +11,7 @@
         <ore name="TantalumOre" spawnsPerChunk="13"/>
         <ore name="RubyOre" spawnsPerChunk="12"/>
     </ores>
-    <explosions damage="0.0" dropBlockChance="1.0"/>
+    <explosions damage="0.2" dropBlockChance="1.0"/>
```

Comment:
This is something we can enable in the client, but should disable server side as it doesn't respect any events and cannot be cancelled.